ocaml/typing/typecore.ml

```diff
@@ -196,16 +196,16 @@ type error =
 exception Error of Location.t * Env.t * error
 exception Error_forward of Location.error
 
-let error_of_filter_arrow_failure ~explanation in_function ty_fun
+let error_of_filter_arrow_failure ~explanation ~first ty_fun
```

Comment:
I do not anticipate interesting changes here from the previous review; did not review. Please let me know if there's a spot I should worry about.

Reply:
That seems right to me.